docker/Dockerfile

```diff
@@ -0,0 +1,121 @@
+ARG ubuntu_version=20.04
+FROM ubuntu:${ubuntu_version}
+
+# Build the Dockerfile in this directory, context one level up
+# docker build -t dyninst -f Dockerfile ../
+
+LABEL maintainer="@hainest,@vsoch"
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/Los_Angeles
+
+# We can use build args to populate the specific versions of dependencies (with defaults here)
+ARG BOOST_VERSION=1.73.0
```

Comment:
Is there a way to read these from base-containers.yaml so that we don't have two places to change the dependency version?

Reply:
This already happens - it's a build argument so it comes from the matrix (what you see there is the default if the build arg is not provided).

Reply:
Ah, ok. I just noticed that when I was looking at the output from the previous run. Good to know!